spec/requests/users_spec.rb

```diff
@@ -1,7 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe "Users", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
-  end
-end
```

Comment:
- [ ] Tests under this file has been deleted. Kindly add them as in the requirement. Let's fix failing tests requested by the previous reviewer instead of deleting them :+1: